tools/azure-sdk-tools/ci_tools/environment_exclusions.py

```diff
@@ -345,6 +349,14 @@
 
 
 def is_check_enabled(package_path: str, check: str, default: bool = True) -> bool:
+    """
+    Single-use function to evaluate whether or not a given check should run against a package.
+
+    In order:
+     - Checks <CHECK>_OPT_OUT for package name.
+     - Honors override variable if one is present: <PACKAGE-NAME>_<CHECK>.
+     - Finally falls back to the pyproject.toml at package root (if one exists) for a tools setting enabling/disabling <check>.
```

Comment:
sorry if dumb - what's the reason for keeping the CHECK_OPT_OUT lists if we have the pyproject.toml? Do they capture different information?

Reply:
Because I haven't scripted out their removal in favor of individual pyproject.toml.